src/doc/python_scripting/functions.rst

```diff
@@ -1849,21 +1774,17 @@ expression : string
     The expression definition as a string.
 
 return type : CLI_return_t
-    The DefineExpression functions return 1 on success and 0 on failure.
+    The DefineTensorExpression function returns 1 on success and 0 on failure.
 
 
 **Description:**
 
-    The DefineTensorExpression creates new tensor variables.
-    Expression variables can be plotted like any other variable.
-    The variableName argument is a string that contains the name of the new
-    variable. You can pass the name of an existing expression if you want
-    to provide a new expression definition.
-    The expression argument is a string that contains the definition of the
-    new variable in terms of math operators and pre-existing variable names
-    Reference the VisIt User's Manual if you want more information on
-    creating expressions, such as expression syntax, or a list of built-in
-    expression functions.
+    DefineTensorExpression creates new tensor variables.
+    Tensor variables define a tensor field over a mesh and are used by the Tensor plot.
+
```

Comment:
There is probably additional detail needed here whe constructing symmetric tensor expressions (as per ordering of terms...its a question users will have) and when constructing from 3 vector variables as opposed to 9 scalars and 2D (4 components) vs 3D (9 components)